src/test/java/com/github/sergdelft/j2graph/representation/EnumTest.java

```diff
@@ -0,0 +1,24 @@
+package com.github.sergdelft.j2graph.representation;
+
+import com.github.sergdelft.j2graph.TestBase;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Tests that exercise the classes with Enums.
+ * This was a bug (issue #3)
+ */
+public class EnumTest extends TestBase {
+
+//    private ClassGraph graph = run("representation/Enum.java");
+
+    //    @TODO - this test fails at the moment. Graph should not be null with enums.
```

Comment:
Is this failing? If so, maybe adding a `@Ignore` or something?

Reply:
And open an issue to it?

Reply:
Yes, it is failing, I will open an issue.

Reply:
Done